server/src/app.ts

```diff
@@ -49,21 +55,25 @@ export const createApp = (): express.Application => {
   app.use(cors());
   app.use(middleware.handle(i18next));
   app.use(express.static(filePath));
+  app.use((req: ReqI18n, res, next) => {
+    const { JWT_SECRET } = process.env;
```

Comment:
Do you think we should use different `JWT_SECRET` here? This may raise a similar `security` problem as you've stated in #180 because there can be a hack to find out the key.

Reply:
I think it's not very important, but it's a great idea to use different JWT secret here. I'll write commit for this.

Reply:
Is bfa966d4a9920856d4a629e23f0ebd4927a712ee the way you intended?

Reply:
Yes but I think putting `number` isn't good idea.
How about `OTHER_SECRET` or `JWT_ETC`?

Reply:
Well, I think `JWT_SECRET_ETC` is better. It's renamed in cf2fc733b75aee984bc6d6b30d67de9da3318c67.